niworkflows/engine/splicer.py

```diff
@@ -0,0 +1,123 @@
+"""Workflow splicing operations."""
+
+import typing as ty
+
+from nipype.pipeline import Workflow
+from nipype.pipeline.engine.base import EngineBase
+
+
+def tag(tag: str) -> 'EngineBase':
+    """
+    Decorator to set a tag on an `init_...wf` function.
+
+    This is used to mark nodes or workflows for replacement in the splicing process.
+    """
+
```

Comment:
This function is unused/untested, and it looks kind of wrong to me, so a test seems critical.